rpc/handlers.go

```diff
@@ -78,6 +83,7 @@ type Handler struct {
 	feederClient  *feeder.Client
 	vm            vm.VM
 	log           utils.Logger
+	ethClient     l1.EthClient
```

Comment:
I think this should be `l1.EthSubscriber`

Reply:
See the above comment (I updated it to use `l1.Subscriber` - no need to give the RPC handler access to everything in `l1.EthSubscriber` imo)